app/src/main/java/woowacourse/movie/ui/setting/SettingPresenter.kt

```diff
@@ -0,0 +1,26 @@
+package woowacourse.movie.ui.setting
+
+import woowacourse.movie.data.alarm.AlarmStateRepository
+import woowacourse.movie.data.reservation.ReservationRepository
+
+class SettingPresenter(
+    private val view: SettingContract.View,
+    private val alarmStateRepository: AlarmStateRepository,
+    private val reservationRepository: ReservationRepository,
+) : SettingContract.Presenter {
+    override fun checkSwitchState() {
+        view.setToggleButton(alarmStateRepository.getData())
+    }
+
+    override fun clickSwitch(isChecked: Boolean) {
+        alarmStateRepository.saveData(isChecked)
+        checkSwitchState()
+
+        val tickets = reservationRepository.getData()
+        if (isChecked) {
+            tickets.forEach { view.setAlarms(it) }
+        } else {
+            tickets.forEach { view.cancelAlarms(it) }
+        }
```

Comment:
```suggestion
        reservationRepository.getData()
            .forEach {
                if (isChecked) {
                    view.setAlarms(it)
                } else {
                    view.cancelAlarms(it)
                }
            }
```
forEach 중복 코드를 줄여보면 어떨까요?